conan/cli/commands/version.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from conan.api.output import cli_out_write
+from conan.cli.command import conan_command
+from conan.cli.formatters import default_json_formatter
+from conan import conan_version
+
+
+def version_text_formatter(version):
+    cli_out_write(f"Conan version {version['version']}")
```

Comment:
Make it a generic ``k: v`` printing

Reply:
Done, but I printed a bit more friendly, like:

```
% conan version
version: 2.0.6
python.version: 3.10.4
python.sys_version: 3.10.4 (main, May 17 2022, 10:53:07) [Clang 13.1.6 (clang-1316.0.21.2.3)]
```